### PR TITLE
Feature/comments

### PR DESCRIPTION
**Pull request by siddharth-vinsol**
* Add Comments to Questions and Indiviual Answers
* Comment can also be upvote/downvote

### CODE REVIEW

File: app/models/answer.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ class Answer < ApplicationRecord
 
   belongs_to :user
   belongs_to :question
+  has_many :comments, as: :commentable, dependent: :restrict_with_error
```
Comment (shubhamjuneja1) [inline app/models/answer.rb]: Can we do this in votehandler?

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
+  before_action :set_commentable, only: [:create]
```
Comment (shubhamjuneja1) [inline app/controllers/comments_controller.rb]: rename to load_commentable

File: app/models/question.rb [hunk 3]
```diff
@@ -5,6 +5,7 @@ class Question < ApplicationRecord
   belongs_to :user
   has_many :answers, dependent: :restrict_with_error
   has_many :sorted_answers, -> { order(Arel.sql('total_upvotes - total_downvotes DESC')) }, class_name: 'Answer', dependent: :restrict_with_error
+  has_many :comments, as: :commentable, dependent: :restrict_with_error
```
Comment (shubhamjuneja1) [inline app/models/question.rb]: from votehandler

File: app/models/answer.rb (hunk 1 above)
Comment (siddharth-vinsol) [reply app/models/answer.rb]: create commentHandler module

**Review by shubhamjuneja1 — APPROVED** [review]

**Pull request merged**